src/pages/TermsOfService.js

```diff
@@ -72,7 +74,7 @@ const TermsOfServicePage = () => {
     }),
     div({ style: { backgroundColor: 'white', borderRadius: 5, width: 800, maxHeight: '100%', padding: '2rem', boxShadow: Style.standardShadow } }, [
       h1({ style: { color: colors.dark(), fontSize: 38, fontWeight: 400 } }, ['Terra Terms of Service']),
-      needToAccept && div({ style: { fontSize: 18, fontWeight: 600 } }, ['Please accept the Terms of Service to continue.']),
+      !noActionNeeded && div({ style: { fontSize: 18, fontWeight: 600 } }, ['Please accept the Terms of Service to continue.']),
```

Comment:
Can we avoid the double negative here? Instead of storing `noActionNeeded`, can we instead create a variable like `actionRequired`?